Tools/autotest/arducopter.py

```diff
@@ -2193,7 +2210,10 @@ def fly_vision_position(self):
         ex = None
         try:
             self.set_parameter("GPS_TYPE", 0)
-            self.set_parameter("EK2_GPS_TYPE", 3)
```

Comment:
This is inconsistent with the other test fix where you retained EK2 compatibility.

Reply:
OK, added in code to allow it to run with EKF2 or EKF3.